deploy/docker-swarm/infra/aws/variables.tf

```diff
@@ -1,4 +1,4 @@
-variable "region" {
+variable "aws_region" {
   default = "eu-west-1"
```

Comment:
I wil change it, although it's not strictly required, because we set it with an environment variable.